src/main/java/uk/gov/hmcts/reform/judicialapi/service/impl/JudicialUserServiceImpl.java

```diff
@@ -398,7 +398,7 @@ private AuthorisationRefreshResponse buildAuthorisationRefreshResponseDto(
 
     private List<String> getRoleIdList(List<JudicialRoleType> judicialRoleTypes) {
         return judicialRoleTypes.stream()
-                .filter(e -> e.getEndDate() == null || !e.getEndDate().isBefore(LocalDateTime.now()))
+                .filter(e -> e.getEndDate() == null || !e.getEndDate().isBefore(LocalDate.now().atStartOfDay()))
                 .map(JudicialRoleType::getTitle).collect(Collectors.toList());
```

Comment:
if we are just comparing the dates, can't we convert `e.getEndDate()` to local date and then compare with  `LocalDate.now()`?

Reply:
Hi @SujeethKumarAri , thanks for the suggestion. I can't find any advantage with this e.getEndDate() conversation to localdate, as well it also has one more date conversion.
Here both give same functional result
e.getEndDate()                                              LocalDate.startTime             result
----------------                                             --------------------          ----------
29/11/2021 08:00:00.                                    29/11/2021 00:00:00.         show in response
29/11/2021 12:00:00                                      29/11/2021 00:00:00           show in response

e.getEndDate() converted as Localdate       LocalDate.now                      result
----------------------------------------      ---------------                   ------------
29/11/2021 00:00:00.                                      29/11/2021 00:00:00        show in response 
29/11/2021 00:00:00                                       29/11/2021 00:00:00        show in response